code/game/objects/items/storage/boxes.dm

```diff
@@ -863,16 +863,77 @@
 	for(var/i in 1 to 6)
 		new /obj/item/ammo_casing/shotgun/beanbag(src)
 
-/obj/item/storage/box/actionfigure
+/obj/item/storage/box/figurines
 	name = "box of action figures"
-	desc = "The latest set of collectable action figures."
+	desc = "The latest set of collectable figurines."
 	icon_state = "box"
 
-/obj/item/storage/box/actionfigure/PopulateContents()
+/obj/item/storage/box/figurines/PopulateContents()
 	for(var/i in 1 to 4)
-		var/randomFigure = pick(subtypesof(/obj/item/toy/figure))
+		var/randomFigure = pick(pick(subtypesof(/obj/item/toy/figure)),
+			pick(subtypesof(/obj/item/toy/prize)),
+			pick((subtypesof(/obj/item/toy/talking) - typesof(/obj/item/toy/talking/codex_gigas))),

```

Comment:
why not leave the fake codex_gigas toy in the prize pool?

Reply:
It's re-added in the pranks pool. This subtype contains the AI, Griffon, Owl, and Codex Gigas. The Codex doesn't fit with the figurine pool so I moved it.